src/components/BookCard/index.tsx

```diff
@@ -10,11 +13,11 @@ type BookCardProp = {
 
 function BookCard({ book }: BookCardProp) {
   return (
-    <div className={styles.book}>
+    <Link to={`${routes.auth.home}/${book.id}`} className={styles.book}>
```

Comment:
Maybe is a good idea to have a constants file with the paths we have in the app. And then use that constant here.